mariadb-ha/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM registry.access.redhat.com/rhscl/mariadb-101-rhel7
+
+USER root
+
+RUN yum clean all && \
+    yum-config-manager --enable rhel-server-rhscl-7-rpms && \
+    yum-config-manager --enable rhel-7-server-optional-rpms && \
+    INSTALL_PKGS="rh-mariadb101-galera which" && \
```

Comment:
When my build runs I get the following:

```
No package rh-mariadb101-galera available.
```

Where should the `rh-mariadb101-galera` package come form?